resources/quality/tinyboy/tinyboy_fabrikator15_normal.inst.cfg

```diff
@@ -30,8 +30,8 @@ layer_height = 0.2
 layer_height_0 = 0.2
 material_bed_temperature = 60
 material_diameter = 1.75
-material_print_temperature = 210
-material_print_temperature_layer_0 = 0
+material_print_temperature = 200
```

Comment:
This temperature is only going to work well for PLA. And since you have specified it in your quality profile, it will override whatever is set in the material itself.

Reply:
Should I remove both lines from the profiles if I want to leave the temperature settings to the material definition?